### ISSUE TITLE
2.0 cleanup needed

### ISSUE THREAD

**Issue by danischroeter** [kamon-io/kamon-system-metrics#26]
The system metrics documentation should be updated to 2.0 or at least reference the updated doc in the kamon page.
Also it might make sense to close all issues regarding cigar etc to have a clean slate...

Btw: why have most metric keys changed from 1.x to 2.x -> seems arbitrary and is a lot of work to adapt for no apparent reason. Is there a mapping table for the changed keys?
Would be nice to keep the metric keys as consistent as possible in the future... 
Example:
host_cpu+tag(measure=used) -> host_cpu_usage or 
jvm_memory+tag(measure=used) -> jvm_memory_used) - I liked the tagged approach before...
Is there a mapping table?

**Comment by ivantopo** [comment]
Hey @danischroeter,

The reason why most metrics changed was that our naming approach was flawed. In Kamon 1.x was the first time we started using tags in all our integrations and we just went sort of crazy with it, more than we should have. When moving to 2.x we tried to make sure that the definition of "what" is being measured is completely encoded in the metric name and the definition of "from where" is being measured is completely encoded in the tags.

There are a few things that helped us test whether the metric/tag were right.. one of them was asking ourselves: if we aggregated all timeseries of the same metrics, would the resulting timeseries be completely useless? In cases like `host.cpu` where the same metric had different tags for free and used it was obviously a bad choice since even though the two measurements come from the same thing, their meanings are really different. Same goes with `jvm.memory` and pretty much anything where we had something and the opposite of that something encoded in tags of the same metric.

Rest assured that we really don't want to make any more breaking changes like that.. it was just a necessary pain to have a consistent behavior.
